File: study_demo/Study_thread/TestDaemon/TestDaemon/ViewController.m
//
//  ViewController.m
//  TestDaemon
//
//  Created by JianweiChen on 2020/2/24.
//  Copyright © 2020 inke. All rights reserved.
//

#import "ViewController.h"
#import "IKTestThread.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    [IKTestThread test];
}


@end

File: DemoLinkKit/DemoLinkKitDemo/ViewController.swift
//
//  ViewController.swift
//  IMChatKitDemo
//
//  Created by 王虎 on 2017/1/10.
//  Copyright © 2017年 hoowang. All rights reserved.
//

import UIKit
import DemoLinkKit


class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        testExports()
    }

    override func touchesBegan(_ touches: Set<UITouch>, with event: UIEvent?) {
        let chatController = DemoChatViewController()
        self.present(chatController, animated: true) { 
            
        }
    }

}

